bridge/bridge.go

```diff
@@ -187,6 +190,11 @@ func (b *Bridge) Sync(quiet bool) {
 }
 
 func (b *Bridge) add(containerId string, quiet bool) {
+	if _, ok := b.dyingContainers[containerId]; ok {
+		log.Println("container, ", containerId[:12], ", is dying, ignoring")
```

Comment:
Kibana search friendly: `Ignoring dying container:`

Reply:
@pnarozny I kept it consistent. Should I change the other places too?

Reply:
is it dying container or stopping container? It is bit confusing to me